content/en/docs/gke/custom-domain.md

```diff
@@ -22,38 +15,24 @@ so, follow the guide to
 
 If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.cloud.goog**, follow these instructions after running `kfctl build`:
 
-1. Replace `hostname` in `kustomize/iap-ingress/base/params.env` with your own domain.
+1. Remove the substitution `hostname` in kptfile.
```

Comment:
Suggestion: 

- "kptfile" -> "the Kptfile" (reference: https://googlecontainertools.github.io/kpt/search/?q=kptfile)

```suggestion
1. Remove the substitution `hostname` in the Kptfile.
```